docs/source/intervention_models/maternal_anemia_intervention/index.rst

```diff
@@ -0,0 +1,248 @@
+designed for at-risk female popualtion (e.g., women during pregnancy, women of 
+reprodictive age, etc.) for averting the burden of maternal anemia in LMICs. 
+It consists of three tiers: 1) Large-scale food fortificaiton (DFS), 2) Supplmentation 
+(BEP and MMS), and 3) treatment (IV iron).
+
+.. image:: intervention_pyramid.png
+
+The scope of our simulation work is to estimate following measures in 4 focal countries:
+ - the size of eligible population at risk for each tier of the pyramid;
+ - the burden of the maternal anemia in terms of deaths and DALYs occurring in 
+   the population at risk within each tier of the pyramid if we assume 
+   intervention is not applicable; 
+ - the impact at each tier on maternal anemia in terms of averted deaths and DALYs 
+   if we assume complete pyramid of interventions are applied.
+
+.. list-table:: Tiered Intervention
+  :header-rows: 1
+
+  * - Tier
+    - Intervention
+    - Product name
+    - Product definition
+    - Target population
+    - Note
+  * - 1
+    - Large-scale food fortification
+    - DFS
+    - 
+    - Women of reproductive age (15 to 49 years)
+    - 
+  * - 2
+    - Supplementation
+    - BEP
+    - 
+    - PLW at ANC service with BMI less than 18.5
+    - 
+  * - 2
+    - Supplementation
+    - MMS
+    - 
+    - PLW at ANC service with BMI greater than or equal to 18.5
+    - 
+  * - 3
+    - Treatment
+    - IV iron
+    - 
+    - Pregnant women in 2nd/3rd trimester with moderate/severe anemia referred 
+      from ANC to higher level facility
+    - 
+
+.. todo::
+
+  Fill out the table above with intervention product definition.
+
+.. list-table:: Affected Outcomes
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Population
+    - Note
+  * - Population mean hemoglobin
+    - Additive
+    - non-PLW
+    - Effects are not stratified by intervention tiers
+  * - Population mean hemoglobin
+    - Additive and multiplicative combined
+    - PLW
+    - Effects are stratified by intervention tiers 
+
+Baseline Coverage Data
+++++++++++++++++++++++
+
+.. note::
+
+  No existing coverage of each intervention tier
+
+
+.. list-table:: Baseline coverage data
+  :widths: 10 10 10 10 10
+  :header-rows: 1
+
+  * - Location
+    - Subpopulation
+    - Coverage parameter
+    - Value
+    - Note
+  * - Bangladesh
+    - 
+    - 
+    - 
+    - 
+  * - Malawi
+    - 
+    - 
+    - 
+    - 
+  * - Nigeria
+    - 
+    - 
+    - 
+    - 
+  * - Pakistan
+    - 
+    - 
+    - 
+    - 
+
+
+Vivarium Modeling Strategy
+--------------------------
+How to estimate maternal anemia burden in Vivarium:
+ 1. Construct population hemoglobin distributions for each location-/age- group;
+ 2. Apply pregnancy adjustment factor to hemoglobin distributions for pregnant 
+    and general populations;
+ 3. Modify hemoglobin distributions based on intervention risk effects;
+ 4. Calculate maternal anemia burden, measured by deaths, YLDS, and YLLs.
```

Comment:
```suggestion
 4. Calculate maternal anemia burden, measured by prevalence and YLDS.
 5. Calculate maternal disorders burden corresponding to the change in population mean hemoglobin levels among PLW
```

Reply:
Actually, on second thought, we might not want to get into the effects of hemoglobin on anemia and maternal mortality on this page. I think that this page should focus just on the effect of the interventions on hemoglobin of the target populations (and how to identify the target populations). We already have documentation on the effects of hemoglobin here https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.html